Bots/TwitterBot.js

```diff
@@ -1,36 +1,35 @@
-"use strict";
-
-var app = require('express')();
-var server = require('http').Server(app).listen(5000);
-var io = require('socket.io')(server);
-var request = require('superagent');
-
-var Twitter = require('twitter');
-var AuthBot = require('models/AuthBot');
-var Que = require('common/Que')();
-
-var slackPostAPI = require('common/makeCredential')('slack');
-var log = require('common/log');
-
-function TwitterBot(credentials, callback){
-  if(!credentials.consumer_key){ throw new Error("Must set consumer_key."); }
-  if(!credentials.consumer_secret){ throw new Error("Must set consumer_secret."); }
-
-  if((!credentials.access_token_key && !credentials.access_token_secret) && !credentials.screen_name){
-    throw new Error("Must set access_token_key and access_token_secret or screen_name.");
+import { Server } from 'http';
+import express from 'express';
+import socketIO from 'socket.io';
+import request from 'superagent';
+import Twitter from 'twitter';
+import AuthBot from 'models/AuthBot';
+import que from 'common/Que';
+import log from 'common/log';
+
+const app = express();
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).